shared/src/main/scala/MiscHelpers.scala

```diff
@@ -188,7 +188,10 @@ object MiscHelpers:
   end unpackHelper
 
   def vyPrint(x: VAny)(using ctx: Context): Unit =
```

Comment:
I'd rather there were a single recursive `def vyToString(x: VAny)(using Context): String` helper for converting stuff to strings than 3 different methods, because `VList#toStringWithContext` becomes harder to implement